_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <!-- potential code for a footer -->
-<div class="footer container">
-    <img src="/assets/images/STEMLOGO.png">
+<div class="footer pt-3">
+    <img src="/assets/images/STEMLOGO.png" style="max-width: 300px;">
```

Comment:
I tend to prefer against inline styles. I would add a class to this image and put in a CSS property for max-width.

Reply:
done